Makefile

```diff
@@ -1,16 +1,22 @@
 .PHONY: builder
 
-image:
-	DOCKER_BUILDKIT=1 docker build -t duckling:latest .
+IMAGE_NAME=us.gcr.io/eddy-travels-global/duckling:5
```

Comment:
having hard-coded 5 here can lead to issues, maybe leave out tagging out of Makefile

Reply:
> having hard-coded 5 here can lead to issues, maybe leave out tagging out of Makefile

Changed back to duckling:latest